src/main/clojure/jarl/eval.clj

```diff
@@ -428,8 +428,8 @@
 (defn eval-builtin-func [name builtin-func args state]
   (log/debugf "executing built-in func <%s> with args: %s" name, args)
   (try
-    (let [arg-list (utils/indexed-map-to-array args)
-          result (apply builtin-func arg-list)]
+    (let [result (builtin-func {:args (utils/indexed-map-to-array args)
+                                      :builtin-context (:builtin-context state)})]
```

Comment:
Would a reformat align these a bit better?

Reply:
This is fixed in the next PR.